index.bs

```diff
@@ -1598,286 +1600,161 @@ A URL for Blob and File reference</h2>
-* A scheme of the sort <code>urn:uuid</code> [[RFC4122]] could be used,
-  though use of this scheme is unprecedented in HTML and JavaScript web applications.
-  The urn:uuid scheme is very generic.
-  URLs in the scheme urn:uuid have the disadvantage of unfamiliarity and inconsistency across the web platform.
-  A new scheme has the advantage of being explicit about what is being referenced.
-  In theory, URLs make no guarantee about what sort of resource is obtained when they are dereferenced;
-  that is left to content labeling and media type.
-  But in practice, the name of the scheme creates an expectation about both the resource
-  and the protocol of the request-response transaction.
-  Choosing a name that clarifies the primary use case--
-  namely, access to memory-resident {{Blob}} resources--
-  is a worthwhile compromise,
-  and favors clarity, familiarity, and consistency across the web platform.
-
-<h3 id="DefinitionOfScheme">
-The Blob URL</h3>
-
-A <dfn export>Blob URL</dfn> is a <a>URL</a>
-with a <a for=url>scheme</a> of <code>blob</code>,
-a <a for=url>host</a> of the [=url/origin=] of the <a>Blob URL</a>,
-a <a for=url>path</a> with one entry comprised of a UUID [[RFC4122]]
-(see [[#ABNFUUID]]),
-and an <a for=url>object</a> of the associated {{Blob}} or {{File}}.
-A Blob URL may contain an optional <a for=url>fragment</a>.
-The Blob URL is serialized as a string according to the <a>Serialization of a Blob URL</a> algorithm.
-
-A <a for=url>fragment</a>, if used,
-has a distinct interpretation depending on the media type of the {{Blob}} or {{File}} resource in question
-(see [[#fragmentDiscussion]]).
-
-<pre>
-  blob = scheme ":" origin "/" UUID [fragIdentifier]
-
-  scheme = "blob"
-
-  ; scheme is always "blob"
-
-  ; origin is a string representation of the Blob URL's [=url/origin=].
-  ; UUID is as defined in <a biblio data-biblio-type=informative>\[RFC4122]</a> and <a href="#ABNFUUID"></a>
-  ; fragIdentifier is optional and as defined in <a biblio data-biblio-type=informative>\[RFC3986]</a> and <a href="#ABNFFragment"></a>
-</pre>
+Note: other specifications, such as [[MEDIA-SOURCE]] extend this scheme to also refer to other types of objects.
+
+<h3 id="url-intro">
+Introduction</h3>
+
+*This section is informative.*
+
+[=Blob URL|Blob (or object) URLs=] are URLs like
+<code>blob:http://example.com/550e8400-e29b-41d4-a716-446655440000</code>.
+This enables integration of {{Blob}}s and {{File}}s with other Web Platform APIs
+that are only designed to be used with URLs, such as the <{img}> element.
+[=Blob URLs=] can also be used to navigate to as well as to trigger downloads of
+locally generated data.
+
+For this purpose two static methods are exposed on the {{URL}} interface,
+{{URL/createObjectURL(blob)}} and {{URL/revokeObjectURL(url)}}.
+The first method creates a mapping from a URL to a Blob,
+and the second method revokes said mapping.
+As long as a mapping exist a Blob  can't be garbage collected,
```

Comment:
"As long as the mapping exists the Blob can't...", "...revoke the URL...", "as soon as the reference..."